csrc/dd_arith.c

```diff
@@ -488,3 +494,245 @@ ddouble tanhq(ddouble a)
     c = sqrtq(adddq(1.0, sqrq(s)));
     return divqq(s, c);
 }
+
+ddouble tanq(ddouble a)
+{
+    if (iszeroq(a))
+        return Q_ZERO;
+
+    ddouble s, c;
+    s = sinq(a);
+    c = cosq(a);
+    return divqq(s, c);
+}
+
+void sincos(const ddouble a, ddouble *sin_a, ddouble *cos_a) {
```

Comment:
@AaronDJohnson  I think I found the bug!

Can you rename this function `sincosq` and change the call site below?

Reply:
This fixes it locally on my computer with Pop!OS. It looks like the tests would pass with ulps=2. This is still a bit strange to me though, because ulps=1 works for a similar environment for me.

How did you find this bug? Nice work!